ci-operator/config/openshift/release/openshift-release-master__ci-4.9-upgrade-from-stable-4.8.yaml

```diff
@@ -3,6 +3,10 @@ base_images:
     name: "4.8"
     namespace: ocp
     tag: openstack-installer
+  openstack-tests:
+    name: "4.10"
```

Comment:
This either needs to match "latest" release there (4.9) or should be removed if the change was not backported there

Reply:
OK here's my plan:
1. Add a build trigger for creating images for all releases: https://github.com/openshift/release/pull/23629
2. Create the actual release branches on the openstack-tests repo
3. change this PR to include images for all respective branches

Do you think this plan makes sense?

Reply:
yeah, that looks good. Perhaps we'd get feedback earlier if we focus on 4.10 for now - and later do the backports and other updates?

Reply:
>if we focus on 4.10 for now

By "focus on 4.10 now", do you mean:
A) merging this as-is first; OR
B) only adding the image to 4.10 jobs

If A), I'd be fine with it.

If B): The goal here is to add an `openstack-test` to every `e2e-openstack` workflow. Which in turn, applies to every release. Thus _I think_ we need to add a valid image to every version first.

Reply:
I think I'd prefer B) first, as in any case we'd have to create subsequent PRs to add this stanza or update tag from 4.10 everywhere - and repeated PRs to add stanza to 4.9/4.8 would look a bit more consistent. However the difference is negligible really (we should link to this discussion in follow up PRs)

Reply:
I have repurposed this PR in an attempt to respond to the concerns you raise.

This PR is now adding the image to `installer/master` only, and chaging one single job to include the relevant step. It also adds the step in the step-registry.

Is this actually a proper solution to the concerns you raised, @vrutkovs ?

Reply:
The test with 4.10 worked well. I am then completing the work for the whole Installer repo.

Reply:
The rehearsal with Installer master worked well.
One way forward could be to merge this and open a new PR applying the same changes everywhere.

What do you suggest, @vrutkovs?